src/Anthropic.Client.Tests/Services/Messages/MessageServiceTest.cs

```diff
@@ -38,6 +42,213 @@ public async Task CreateStreaming_Works()
         }
     }
 
+    [Fact]
+    public async Task CreateStreamingAggregation_WorksNoContent_RawMessageStartEvent()
+    {
+        // arrange
+        var messagesServiceMock = new Mock<IMessageService>();
+        static async IAsyncEnumerable<RawMessageStreamEvent> GetTestValues()
+        {
+            yield return new(new RawMessageStartEvent(GenerateStartMessage()));

```

Comment:
Just an FYI that we have a codegen change in the pipeline that will add implicit operators from union variant to union, so you will most likely be able to omit many `new(...)` wrappings like this one (and e.g. `Content = new("")` below could be just `Content = ""`)